org-code-javabuilder/lib/src/main/java/dev/javabuilder/WebSocketServer.java

```diff
@@ -73,6 +73,7 @@ public void onOpen(Session session) {
     this.logger.setUseParentHandlers(false);
 
     Properties.setConnectionId(connectionId);
+    Properties.setIsDashboardLocalhost(true);
```

Comment:
Do we want to set this here? The localhost websocket server should be able to fetch image assets from localhost dashboard

Reply:
So this won't actually affect the localhost websocket server since it uses `LocalContentManager` which will always try to fetch the asset URL - the stubbing logic is only in `AWSContentManager`

Reply:
Thinking out loud a bit here. Once we get outside the WebSocketServer class or LambdaRequestHandler class, it's a bit unclear what this property gets used for. The property is set globally, so in the future, I could see someone using it in some of the shared code, expecting it to consistently indicate that dashboard is inaccessible. But that wouldn't be the case for requests originating from the local WebSocket server. Perhaps naming this something like `InaccessibleDashboard` or `DashboardReachable` and then setting it accordingly would clarify intent.

Reply:
Hmm (also thinking out loud here) I think my worry with this is that "inaccessible" is a state derived from certain combinations of dashboard and Javabuilder, whereas whether the request originated from localhost dashboard is only dependent on dashboard. We have:
localhost Dashboard + websocket server = accessible
localhost Dashboard + deployed Javabuilder = inaccessible
adhoc/staging/prod dashboard + deployed Javabuilder = accessible
adhoc/staging/prod dashboard + websocket server (would we ever have this scenario?) = accessible

I guess the question is, does that logical reduction belong in the handler/Properties? Currently that logic is encapsulated in AWSContentManager. I could go either way on this - do we think it's more useful to store this accessible/inaccessible state in Properties, or will knowing that the request originated from localhost dashboard be useful by itself for other situations with different constraints?

Reply:
Hmm maybe this is more of a special one-time-use case. I can't off the top of my head think of another situation where we'd want to know whether or not the request came from localhost once we're on the buildAndRun Lambda/WebSocket server. As a side note, I don't _think_ we'd ever set up that 4th scenario you listed. It'd be too much work without much gain.

I'm ok with whichever option you end up going with. To throw one more suggestion in the mix: Perhaps just leaving the property unset from the WebSocket server would solve for both our concerns. Then, if we get a future scenario where having this detail on the WebSocket server would be useful, we can just add it, but we also are less likely to add it without a more full understanding of what it actually does.

Reply:
Gotcha, that makes sense. I think in that case I'll err on the side of being descriptive here and call this something like `canAccessDashboardAssets` so its intended use is clear and that way we can also leave it unset in the WebSocketServer.